### PR TITLE
Azure Device Twin and RBAC docs

### PR DESCRIPTION
**Pull request by estenberg**
Signed-off-by: Eystein Måløy Stenberg <eystein.maloy.stenberg@northern.tech>

Note that the RBAC docs does not seem to be quite correct, at least not yet (https://tracker.mender.io/browse/MEN-5386).

### CODE REVIEW

**Comment by oleorhagen** [comment]
Remember this needs a cherry to 3.2.x

**Comment by estenberg** [comment]
May need to change "Manage" to "Deploy" in the last sentence, depending on https://tracker.mender.io/browse/MEN-5386.

**Comment by oleorhagen** [comment]
> May need to change "Manage" to "Deploy" in the last sentence, depending on https://tracker.mender.io/browse/MEN-5386.

5386 is in now

File: 08.Server-integration/04.Azure-IoT-Hub/docs.md [hunk 1]
```diff
@@ -4,13 +4,16 @@ taxonomy:
     category: docs
 ---
 
-Mender supports preparing devices to use Azure IoT Hub. Assuming you already have Mender in place on the device, it is automatically created in Azure IoT Hub and applications running on the device get access to the Azure IoT connection string. With Mender in place, you can simply start writing your Azure IoT application by reading the connection string from a well-defined place in the device. This integration is available in all Mender plans, as well as Mender Open Source.
+Mender supports preparing devices to use Azure IoT Hub. Assuming you already have Mender in place on the device, it is automatically created in Azure IoT Hub and applications running on the device get access to the Azure IoT connection string. With Mender in place, you can simply start writing your Azure IoT application by reading the connection string from a well-defined place in the device. Secondly, the Device Twin from the Azure IoT Hub becomes available to manage from Mender.
```
Comment (oleorhagen) [inline 08.Server-integration/04.Azure-IoT-Hub/docs.md:7]:
```suggestion
Mender supports preparing devices to use Azure IoT Hub. After Mender is installed on the device, it is automatically created in Azure IoT Hub, and applications running on the device get access to the Azure IoT connection string. With Mender in place, you can simply start writing your Azure IoT application by reading the connection string from a well-defined place on the device. Secondly, the Device Twin from the Azure IoT Hub becomes available to manage from Mender.
```

**Review by oleorhagen — APPROVED** [review]

**Comment by estenberg** [comment]
Our CI system has something against me personally I think... Now...?

**Comment by estenberg** [comment]
3.2.x cherry pick: 08cab03476449af237608be4e1da060bd0e56a0f
hosted does not seem to have been rebased for 3.2 yet, skipped.

**Pull request merged**